### PR TITLE
Update info endpoint usage on frontend

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Update info endpoint usage on frontend

### CODE REVIEW

File: frontend/app/src/mixins/backend-mixin.ts [hunk 1]
```diff
@@ -32,10 +31,6 @@ export default class BackendMixin extends Vue {
   async mounted() {
     await this.load();
     this.loaded();
-    const loglevel = this.options.loglevel;
```
Comment (kelsos) [inline frontend/app/src/mixins/backend-mixin.ts:35]: after removing this will the options loglevel still be considered on the electron app?

Comment (kelsos) [reply frontend/app/src/mixins/backend-mixin.ts:35]: Ok it seems to work fine

Comment (kelsos) [reply frontend/app/src/mixins/backend-mixin.ts:35]: @lukicenturi so the UI updates properly if I pass a config file [loglevel](https://rotki.readthedocs.io/en/latest/usage_guide.html#set-the-backend-s-arguments)

image

but frontend remains to WARN
image

We still need to keep this behavior when we are packaged using electron. When there is a config file that sets loglevel, the loglevel from the config file should take priority and cannot be modified.

Also in electron the user can modify the loglevel and this takes priority, so at least for electron we shhou keep this behavior here and only set from the one in the backend in docker

Comment (lukicenturi) [reply frontend/app/src/mixins/backend-mixin.ts:35]: I guess no. The options log level is used to restart the backend. And the log level value is always taken from the API at `store.ts`. So I think they will be always the same(?) @kelsos

**Review by kelsos — APPROVED** [review]
lgtm, thank you.

It seems that what I did originally was partially wrong, since it used the wrong loglevel function and it didn't update properly.

I changed the approach a bit and pushed a commit on top, also tested with both docker and the packaged application and everything seems to work fine.

**Pull request merged**